chore(release notes) Changelog for Cloud cycle38 (#1041)
seqeralabs/docs: changelog/seqera-cloud/v26.1.0_cycle38.md

- Added admin endpoint `GET /api/admin/orgs/{orgId}/roles` to list organization roles.
- Added endpoint `GET /orgs/{orgId}/users/{userId}/roles` to list user roles in organization.
- Added `maxCustomRolesPerOrg` quota definition and enforcement, limiting the number of custom roles per organization.

### Data Explorer

- Fixed file preview functionality for files containing special characters (brackets) in filenames by properly encoding and decoding file paths.

### General

- Updated container `max-width` from 1120px to 1672px for improved layout readability on data-dense pages.
- Improved error messaging when attempting to delete labels that cannot be removed.
- Updated DRAGEN compute environment helper text to recommend f2 instances instead of deprecated f1 instances.

## Bug fixes

- Fixed **Open in new tab** functionality for workflow reports by implementing a separate route with report path passed as query parameter.
- Fixed step status indication mechanism in launch forms to properly track all controls registered to a section, including those registered after initial creation.
- Fixed Data Explorer path decoding to be null-safe, preventing test failures.

> - Fixed an issue where custom pipeline info fetch error messages were incorrectly removed when selecting the same value again in pipeline, revision, or main script fields.

Reviewer: ```suggestion
- Fixed an issue causing custom pipeline info fetch error messages to be incorrectly removed when selecting the same value again in pipeline, revision, or main script fields.
```
